.github/workflows/ci.yml

```diff
@@ -13,13 +13,19 @@ jobs:
       uses: actions/setup-java@v1
       with:
         java-version: 1.8
+    - uses: actions/cache@v1
```

Comment:
Did this improve compilation times, do you have any sense of the improvements?

Reply:
I think it did. This [branch](https://github.com/velocidi/velocidi-android-sdk/actions?query=branch%3Aupdate-ci++) as lower the run times than [master](https://github.com/velocidi/velocidi-android-sdk/actions?query=branch%3Amaster++). 

The major improvement was in `Build with Gradle` step. Previously, it took ~2:50 sec to run. Now it takes ~1:30 sec.